### PR TITLE
vaktija-desklet@MuxBH28: Initial Commit

### PR DESCRIPTION
**Pull request by MuxBH28**

### CODE REVIEW

File: vaktija-desklet@MuxBH28/README.md
Comment (rcalixte) [inline vaktija-desklet@MuxBH28/README.md]: Users will be able to install locally once this is merged.

File: vaktija-desklet@MuxBH28/files/vaktija-desklet@MuxBH28/desklet.js
Comment (rcalixte) [inline vaktija-desklet@MuxBH28/files/vaktija-desklet@MuxBH28/desklet.js:29]: Take a look at other applets for how to implement proper support for translatable strings. Ask if you have any questions.

File: vaktija-desklet@MuxBH28/files/vaktija-desklet@MuxBH28/metadata.json
Comment (rcalixte) [inline vaktija-desklet@MuxBH28/files/vaktija-desklet@MuxBH28/metadata.json]:
```suggestion
    "version": "1.1",
    "author": "MuxBH28",
    "website": "https://github.com/MuxBH28/vaktija-desklet",
    "description": "This desklet fetches and displays prayer times using data from vaktija.ba API.",
    "credits": "Desklet created by MuxBH28 - https://github.com/MuxBH28/ \nAPI created by vaktija.ba - https://github.com/vaktija"
```

File: vaktija-desklet@MuxBH28/files/vaktija-desklet@MuxBH28/metadata.json
Comment (rcalixte) [inline vaktija-desklet@MuxBH28/files/vaktija-desklet@MuxBH28/metadata.json]:
```suggestion
```

**Comment by rcalixte** [comment]
Run validation locally on your branch with `./validate-spice vaktija-desklet@MuxBH28` from the root of your repository so that you can check for errors. Feel free to ask if you have any questions as well.

File: vaktija-desklet@MuxBH28/files/vaktija-desklet@MuxBH28/desklet.js
Comment (MuxBH28) [reply vaktija-desklet@MuxBH28/files/vaktija-desklet@MuxBH28/desklet.js:29]: This is my first desklet and right now it works as expected but of course I am gonna be improving it.
Almost always I am using other desklets as a go to when fixing something.
If you have some more documentations (apart from cinnamon official), feel free to share with me please.

I will probably need your help with some things, especially with settings scheme which I could not figure out (usually [TypeError: this.settings.get_int is not a function])

Thank You

Comment (rcalixte) [reply vaktija-desklet@MuxBH28/files/vaktija-desklet@MuxBH28/desklet.js:29]: > If you have some more documentations (apart from cinnamon official), feel free to share with me please.

At the moment, I would refer to the other desklets, especially if there is one that implements a feature you would like to implement yourself.



> I will probably need your help with some things, especially with settings scheme which I could not figure out (usually [TypeError: this.settings.get_int is not a function])

Feel free to ask and we can work through it together. I can link you to code that exists already that works that would hopefully get you unblocked.



> This is my first desklet

This is fine. Just remember that you don't need to rush.

File: vaktija-desklet@MuxBH28/README.md
Comment (MuxBH28) [reply vaktija-desklet@MuxBH28/README.md]: Fixed it. Just waiting for merge now

Comment (rcalixte) [reply vaktija-desklet@MuxBH28/README.md]: Do you need a hand with translation support?

**Review by rcalixte — APPROVED** [review]

File: vaktija-desklet@MuxBH28/README.md
Comment (MuxBH28) [reply vaktija-desklet@MuxBH28/README.md]: My desklet is location based, in the future I am planning to translate it to English too (not much text tbh).
Right now my priority is to get settings scheme working. I am gonna play around it these few days and get back to you.

Thanks again!

**Pull request merged**
